### PR TITLE
Lesson5

### PR DESCRIPTION
**Pull request by ameyunaikgithub**

### CODE REVIEW

File: ajax/public/index.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    Name:<input type = 'text' id = 'username'/>
+    Comment: <input type='text' id='comment'/>
+    submit: <input type = 'submit' id= 'submit'/>
+    <script>
+
+const a = document.getElementById('submit');
+a.onclick=()=> {
+  const name= document.getElementById('username').value;
+  const comment = document.getElementById('comment').value;
+  //  alert(name + ' ' + comment);
```
Comment (songz) [inline ajax/public/index.html]: remove comments.

File: chatroom/chatroom.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const fs = require("fs");
+const app = express();
+app.listen(3614);
+app.use(express.static('public'));
+app.get('/XMLHttpRequest', (req,res) =>{
+    fs.appendFile('/home/amey/garageScript/chatroom/public/data.txt',`/n name: ${req.query.Name} comment: ${req.query.Comment}`);
+    res.send("your comment have been saved");
+    });
```
Comment (songz) [inline chatroom/chatroom.js]: indentation

File: chatroom/public/index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    <meta charset="UTF-8">
+    <title></title>
+  </head>
+  <body>
+    Name:<input type="text" id="Name" />
+    Comment:<input type="text" id="Comment" />
+    <input type="submit" id="button" />
+    <div>
+      <pre id="chatid"/>
+                </div>
```
Comment (songz) [inline chatroom/public/index.html]: INDENT

File: frontend/public/data.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+hello amey shoes
```
Comment (songz) [inline frontend/public/data.txt]: don't commit generated files.

File: ajax/public/index.html [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    <meta charset="UTF-8">
+    <title></title>
+  </head>
+  <body>
+    Name:<input type = 'text' id = 'username'/>
+    Comment: <input type='text' id='comment'/>
+    submit: <input type = 'submit' id= 'submit'/>
+    <script>
+
+const a = document.getElementById('submit');
```
Comment (saumya665) [inline ajax/public/index.html]: user a better variable name than a

File: ajax/public/index.html [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+a.onclick=()=> {
+  const name= document.getElementById('username').value;
+  const comment = document.getElementById('comment').value;
+  const b = new XMLHttpRequest();
```
Comment (saumya665) [inline ajax/public/index.html]: use more meaningful variable name than b

**Review by saumya665 — APPROVED** [review]

**Review by MadhuBeesetty — APPROVED** [review]

**Pull request merged**